omero/developers/Tables.rst

```diff
@@ -295,13 +295,49 @@ These are only an introduction to using OMERO.tables and do not show its full
 potential, see :ref:`tables-going-forward` for some inspiration.
 
 
+Data viewing in OMERO.web
+-------------------------
+
+OMERO.web can display table data in a separate tab (attachment link) or in the image context, given the following setup:
+
+::
+
+    Dataset <- Table (Attachments panel > link)
+       |- Image 1 <- (Table panel > link, row values)
+       |- Image 2 <- (Table panel > link, row values)
+       |- ...
+
+The rows of the table will be rendered for each image of the dataset. The screenshot bellow shows the right-hand side panel of a selected Image in OMERO.web.
```

Comment:
typo: "below".
Also nicer to use "Image" and "Dataset" instead of "image" and "dataset". Same with sentence above.